XIVComboExpanded/CustomComboPreset.cs

```diff
@@ -1587,16 +1592,21 @@ public enum CustomComboPreset
     [CustomComboInfo("Prioritize Sacrificium over Lemure's", "Replace Lemure's Slice/Scythe with Sacrificium even when you have 2 Void Shroud.  This can help ensure you do not miss a usage of Sacrificium, at the cost of potentially missing a Lemure's Slice/Scythe usage.", RPR.JobID)]
     ReaperLemuresSacrificiumPriorityFeature = 3941,
 
-    [IconsCombo([RPR.ArcaneCircle, UTL.ArrowLeft, RPR.PlentifulHarvest])]
+    [IconsCombo([RPR.ArcaneCircle, UTL.ArrowLeft, RPR.PlentifulHarvest, UTL.Blank, RPR.Buffs.PerfectioParata, UTL.Checkmark])]
```

Comment:
uhhhhhh is perfectiopara related to plentiful harvest or did you just c/p and forget to change the buff name? lmao

Reply:
oh that's because of the new feature
but then this one line shouldn't have it

Reply:
I fixed it and replaced by `RPR.Buffs.ImmortalSacrifice`

Reply:
Derp, yep, copy-pasta fail on my part.  Good catch XD.